Support for react > v17?

After starting a new project and trying to install useWallet, I get this error:
```
npm ERR! While resolving: project-frontend@0.1.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.13.1" from use-wallet@0.13.1
npm ERR! node_modules/use-wallet
npm ERR!   use-wallet@"*" from the root project

```

It would be great if the NPM package.json can be updated to reflect the newer version of react (17.x) as well.

--- a/package.json
+++ b/package.json
@@ -67,7 +67,7 @@
     "typescript": "^4.0.2"
   },
   "peerDependencies": {
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1"
+    "react": "^16.13.1 || ^17",
+    "react-dom": "^16.13.1 || ^17"
   }
 }

